__tests__/app.test.js

```diff
@@ -47,3 +47,26 @@ describe("/api", () => {
       });
   });
 });
+
+describe("/api/articles", () => {
+  test("should return all articles in an array of objects", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.articles.length).toBe(13);
+        response.body.articles.forEach((article) => {
+          expect(article).toMatchObject({
+            article_id: expect.any(Number),
+            title: expect.any(String),
+            topic: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String)
+          });
+        });
+      });
```

Comment:
You should have a test checking that the array is sorted by created_at and desc order :) (use jest-sorted!)